src/locales/en/header.json

```
@@ -0,0 +1,12 @@
+{
+  "home": "Home",
+  "about": "About",
+  "contact": "Contact",
+  "login": "Login",
+  "wish list": "Wish list",
+  "my cart": "My cart",
```

```suggestion
  "wish_list": "Wish list",
  "my_cart": "My cart",
```
Key thì mình k để lại khoảng trống nha

---

Dạ vâng anh